notebooks/data-sources/TestGrid/metrics/test_pass_failures.ipynb

```diff
@@ -2,91 +2,143 @@
  "cells": [
```

Comment:
<p>Do you still need #noqa here?</p><p> </p><pre class="ql-syntax" spellcheck="false">from ipynb.fs.defs.metric_template import (&nbsp;# noqa: E402
&nbsp;&nbsp;testgrid_labelwise_encoding,
&nbsp;&nbsp;CephCommunication,
&nbsp;&nbsp;save_to_disk,
&nbsp;&nbsp;read_from_disk,
)&nbsp;# noqa: E402
</pre><p> </p><p>I think this cell will pass pre-commit. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/200/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
updated :+1: